### DISCUSSION TITLE
LYWSD03MMC Previously flashed with ATC

### DISCUSSION THREAD

**Discussion by Spookster** [pvvx/ATC_MiThermometer#452] [Q&A]
I have a few LYWSD03MMC that I wish to convert to zigbee, they have previously been flashed with https://github.com/atc1441/ATC_MiThermometer

When I open the link here via pvvx (android phone) the flasher detects ATC_xxxxx and pairs with it.
It detects the alternate firmware 

![Screenshot_20231226_100034_Chrome](https://github.com/pvvx/ATC_MiThermometer/assets/982535/8415340b-5031-4b88-aec3-0082e5ac694a)

I click on TelinkOTA, but which firmware do I need to upload for flashing?

![Screenshot_20231226_095527_Chrome](https://github.com/pvvx/ATC_MiThermometer/assets/982535/452629fe-0952-435b-9de2-3946548c3756)

If not possible to flash directly from here, which steps do I need to follow?

As an aside - I literally threw 10 ble sensors in the bin as Ive already changed them out for zigbee sensors, when this tutorial popped up in my feed, so I dumpster dived and retrieved them :p

**Comment by pvvx** [comment]
[README.md](https://github.com/pvvx/ATC_MiThermometer)
Alternative firmware and configurator from ATC1441 (variant ATC_Thermometer.bin) can be found here: [atc1441/ATC_MiThermometer](https://github.com/atc1441/ATC_MiThermometer).

Use file "[ATC_v46.bin](https://github.com/pvvx/ATC_MiThermometer/raw/master/ATC_v46.bin)"

[To flash the Zigbee firmware, use a Google Chrome, Microsoft Edge or Opera Browser.](https://github.com/pvvx/ZigbeeTLc?tab=readme-ov-file#to-flash-the-zigbee-firmware-use-a-google-chrome-microsoft-edge-or-opera-browser)
Paragraph 5.
Alternatively you can choose a specific firmware binary (i.e. the original firmware) via the file chooser. Example: In the file name field paste "https://github.com/pvvx/ZigbeeTLc/raw/master/bin/1141-020a-01173001-Z03MMC.zigbee".
